web/src/components/Navigation/HeaderLink.vue

```diff
@@ -1,13 +1,13 @@
 <template>
   <div class="p-4 m-4">
     <nuxt-link :to="getLink">
-      <div class="h-36 w-36 items-center">
-        <Sofa v-if="link === 'The Basement'" />
-        <Controller v-else-if="link === 'Games'" />
-        <Burger v-else-if="link === 'Meals'" />
-        <Mug v-else-if="link === 'Drinks'" />
-        <Reel v-else-if="link === 'Movies'" />
-        <User v-else-if="link === 'auth'" />
+      <div class="items-center text-center">
+        <Sofa v-if="link === 'The Basement'" class="block m-auto" />
+        <div v-else-if="link === 'Games'">Games</div>
+        <div v-else-if="link === 'Meals'">Meals</div>
+        <div v-else-if="link === 'Drinks'">Drinks</div>
+        <div v-else-if="link === 'Movies'">Movies</div>
+        <User v-else-if="link === 'auth'" class="block m-auto" />
```

Comment:
I think we can simplify this now that each link is text.

## Before
```vue
  <div class="p-4 m-4">
    <nuxt-link :to="getLink">
      <div class="items-center text-center">
        <Sofa v-if="link === 'The Basement'" class="block m-auto" />
        <div v-else-if="link === 'Games'">Games</div>
        <div v-else-if="link === 'Meals'">Meals</div>
        <div v-else-if="link === 'Drinks'">Drinks</div>
        <div v-else-if="link === 'Movies'">Movies</div>
        <User v-else-if="link === 'auth'" class="block m-auto" />
      </div>
    </nuxt-link>
    <!-- <nuxt-link v-else class="text-3xl m-2" :to="getLink">{{ link }}</nuxt-link> -->
  </div>
 ```
 
 ![image](https://user-images.githubusercontent.com/4812409/113957589-80454a00-97ed-11eb-9cfb-7a632a6c629a.png)
 Notice there is no green (padding) around the icon, which means the clickable space is 24px x 24px. 

 ## After
```vue
<template>
  <nuxt-link class="p-4 m-4 block" :to="getLink">
    <Sofa v-if="link === 'The Basement'" />
    <User v-else-if="link === 'auth'" />
    <span v-else>{{ link }}</span>
  </nuxt-link>
</template>
```
![image](https://user-images.githubusercontent.com/4812409/113957515-5ab84080-97ed-11eb-88e8-661572dc508c.png)

Moving the block styling to the `a` tag gives it the ability to have padding (the green box in the image below), and replicates what we were doing with `<div class="p-4 m-4">`. The `a` tag itself, along with it's children are what respond to click events, so this helps us get to the 44px height suggested for thumbs if we were on an iPad.